### DISCUSSION TITLE
Quay.io support for repository metadata

### DISCUSSION THREAD

**Discussion by startxfr** [artifacthub/hub#2090] [General]
When submitting 

```bash
oras push   quay.io/startx/fedora:artifacthub.io  \
 --manifest-config /dev/null:application/vnd.cncf.artifacthub.config.v1+yaml  \
 artifacthub-repo.yml:application/vnd.cncf.artifacthub.repository-metadata.layer.v1.yaml
```

I get in return 

```bash
Uploading 2a46325a484f artifacthub-repo.yml
Error: failed commit on ref "unknown-sha256:e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855": unexpected status: 500 Internal Server Error
```

Same for docker.io ([wich is not already supported](https://github.com/docker/roadmap/issues/135)).

Witch repository should i use to get a [verified image](https://artifacthub.io/docs/topics/repositories/#repository-metadata) ?

**Comment by tegioz** [comment]
Hi @startxfr 👋🏻

Even though quay.io supports OCI artifacts, it looks like only some OCI types are enabled at the moment:

> By default, only Helm charts, cosign (artifact signing), and ztsd based compression are the enabled OCI types enabled. (From: https://cloud.redhat.com/blog/quay-oci-artifact-support-for-helm-charts).

The Oras site has a [list of registries](https://oras.land/implementors/) that support OCI artifacts, hope this helps :)
